### PR TITLE
Feature/issue 165/user settings

### PR DESCRIPTION
**Pull request by yousfino**
Created the following pages:
- Settings Page - accessed through user profile.
- Notifications Page - accessed through Settings Page.
- Help Page - accessed through Settings Page.
- Language Page - accessed through Settings Page.
- Logout through Settings Page.

### CODE REVIEW

**Review by KhalilGarali — CHANGES_REQUESTED** [review]
Other than the comments I made, everything looks good!

File: ClientApp/app/(tabs)/profile/index.tsx [hunk 1]
```diff
@@ -155,7 +158,7 @@ const ProfilePage: React.FC = () => {
             <View className="flex-row justify-center items-center p-4 relative">
                 <Text className="text-2xl font-bold text-black">Profile</Text>
                 <TouchableOpacity 
-                    onPress={() => console.log("Edit Profile Pressed")} 
+                    onPress={() => router.push('/(tabs)/profile/(settings)/settingsPage')} 
                     className="absolute right-4" // Position Edit button in top-right
                 >
                     <Text className="text-blue-600 font-semibold">Edit</Text>
```
[the remark this answers is not in the document]
Comment (yousfino) [reply ClientApp/app/(tabs)/profile/index.tsx:164]: done! thank you

File: ClientApp/app/(tabs)/profile/_layout.tsx [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import { Stack } from 'expo-router';
+
+export default function ProfileLayout() {
+  return <Stack screenOptions={{ headerShown: true }} />;
```
[the remark this answers is not in the document]
Comment (yousfino) [reply ClientApp/app/(tabs)/profile/_layout.tsx:4]: resolved!

File: ClientApp/app/(tabs)/profile/(settings)/settingsPage.tsx [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+        </TouchableOpacity>
+
+        {/* Help */}
+        <TouchableOpacity style={styles.option} onPress={() => router.push('/profile/(settings)/helpPage')}>
+          <Ionicons name="help-circle-outline" size={mvs(24)} color="black" style={styles.icon} />
+          <Text style={styles.text}>Help</Text>
+        </TouchableOpacity>
+      </ScrollView>
+
+      {/* Fixed Logout Button */}
```
[the remark this answers is not in the document]
Comment (yousfino) [reply ClientApp/app/(tabs)/profile/(settings)/settingsPage.tsx:38]: what a great idea! done done!

**Review by KhalilGarali — APPROVED** [review]
Nice! Thank you!

**Review by NicolasChelico — APPROVED** [review]
good work

**Pull request merged**